assets/js/urlBuilder.js

```diff
@@ -0,0 +1,13 @@
+const allUrlToBuild = document.getElementsByClassName('js-build-url');
+
+if (allUrlToBuild.length) {
+    const currentUrl = new URL(window.location.href);
+
+    [...allUrlToBuild].forEach(form => {
+        const targetUrl = form.dataset.action;
```

Comment:
why do you use data-action, and not directly the action? 
It would be more more accessible I guess

Reply:
It's because everyone knows when they see an element with a dataset that this element will be manipulated with Javascript, which is not clear if I directly use the `action` attribute.

You could argue that this is already pretty clear that this will be manipulated by Javascript thanks to the `js-build-url` class, and you'd be right.

And I could also be wrong on the reason I gave, I don't know

Reply:
If people don't have js, if you use action only, it degrades really smooothly